stylus/base/helpers.styl

```diff
@@ -101,61 +109,71 @@ for $alignment, $text-align in $alignments
 
 for $name, $pair in $colors
   $color = $pair[0]
+
   .has-text-{$name}
     color: $color !important
+
   a.has-text-{$name}
     &:hover
     &:focus
       color: sass-darken($color, 10%) !important
+
   .has-background-{$name}
     background-color: $color
 
 for $name, $shade in $shades
   .has-text-{$name}
     color: $shade !important
+
   .has-background-{$name}
     background-color: $shade !important
 
-.has-text-weight-light
-  font-weight: $weight-light !important
-.has-text-weight-normal
-  font-weight: $weight-normal !important
-.has-text-weight-semibold
-  font-weight: $weight-semibold !important
-.has-text-weight-bold
-  font-weight: $weight-bold !important
+$weights ?= light normal semibold bold
```

Comment:
The `light` is a built-in function of Stylus, it renders nothing when iterating it. It must be 'quoted'.

```stylus
.has-text-weight-light {
  font-weight: !important
}
.has-text-weight-normal {
  font-weight: 400 !important
}
```